src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -252,16 +249,99 @@ private void Method()
             return AssertCodeCleanupResult(expected, code);
         }
 
-        protected static async Task AssertCodeCleanupResult(string expected, string code, bool systemUsingsFirst = true, bool separateUsingGroups = false)
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.CodeCleanup)]
+        public Task PlaceUsingDirectivesOutsideNamespace()
+        {
+            var code = @"namespace A
+{
+    using System;
+
+    internal class Program
+    {
+        private void Method()
+        {
+            Console.WriteLine();
+        }
+    }
+}
+";
+
+            var expected = @"using System;
+
+namespace A
+{
+    internal class Program
+    {
+        private void Method()
         {
+            Console.WriteLine();
+        }
+    }
+}
+";
+
+            return AssertCodeCleanupResult(expected, code);
+        }
+
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.CodeCleanup)]
+        public Task PlaceUsingDirectivesInsideNamespace()
```

Comment:
💡 This should include additional tests:

1. Using directives start outside the namespace, with `placeUsingsOutsideNamespace: true`
1. Using directives start outside the namespace, with `placeUsingsOutsideNamespace: false`
1. A pair of tests that start with usings _both_ inside and outside the namespace (one file has multiple using directives), with each of the placement options

Reply:
Done.